tests/qnodes/test_qnode_passthru.py

```diff
@@ -0,0 +1,186 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the :mod:`pennylane` :class:`.PassthruQNode` class.
+"""
+import pytest
+import numpy as np
+
+try:
+    import tensorflow as tf
+
+    if tf.__version__[0] == "1":
+        tf = None  # default.tensor.tf requires TF 2
+    else:
+        from tensorflow import Variable
+except ImportError:
+    tf = None
+
+import pennylane as qml
+from pennylane.qnodes import PassthruQNode, BaseQNode, JacobianQNode
+
+
+
+@pytest.fixture(scope="function")
+def mock_qnode(mock_device):
+    """Simple PassthruQNode with default properties."""
+
+    def circuit(x):
+        qml.RX(x, wires=0)
+        return qml.expval(qml.PauliZ(0))
+
+    node = PassthruQNode(circuit, mock_device)
+    return node
+
+
+class TestPassthruBasics:
+    """Tests basic PassthruQNode properties."""
+
+    def test_always_mutable(self, mock_qnode):
+        """PassthruQNodes are always mutable."""
+        assert mock_qnode.mutable
+
+    def test_repr(self, mock_qnode):
+        """String representation."""
+        assert repr(mock_qnode) == "<PassthruQNode: device='mock_device', func=circuit, wires=2>"
+
+
+@pytest.mark.skipif(tf is None, reason="TensorFlow 2.0 not found.")
```

Comment:
Is this decorator doing anything? It seems to be marking a fixture, not a test or test class

Reply:
This `skipif` will apply to any test function or class that uses the fixture :smile: Got to love pytests dependency injection